content/docs/command-reference/push.md

```diff
@@ -151,11 +151,11 @@ $ dvc push data.zip.dvc
 ## Example: With dependencies
 
 Demonstrating the `--with-deps` option requires a larger example. First, assume
-a [pipeline](/doc/command-reference/pipeline) has been setup with these
+a [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
+$ dvc dag
```

Comment:
@jorgeorpinel Where can I find a remote for this workspace? Clearly I will need to create a workspace to get the correct `dvc dag` output for this pipeline. But I don't see any repo defined in this doc.

Reply:
It's pre-configured in the https://github.com/iterative/example-get-started project, you shouldn't need to do anything special, just clone it.

Reply:
But honestly `dvc dag` is not going to have a useful output here, it will be too long. Just use `cat dvc.yaml` (and it's output) instead please.

Reply:
Actually... cat dvc.yaml won't be a good idea, it will be way too long. Maybe for now just replace this whole block for just a list of stage names in the previous paragraph. Maybe the DVC-file names (without .dvc) for now...

Reply:
Notice that we shouold use the `test-posts` and `matrix-train` stage names here, to match https://github.com/iterative/dvc.org/pull/1591/files

Reply:
I added `test-posts` and `matrix-train` stage names in the output (straight, without any commands).
Since this was a hypothetical workspace, I could only figure out these 2 stage names from the example updates in #1591.
Let me know if I missed anything.

Reply:
Moved this conversation to https://github.com/iterative/dvc.org/pull/1584#pullrequestreview-454571458. Resolving here.